### ISSUE TITLE
References to `@storybook/addon-docs/blocks` should now just use `@storybook/addon-docs` to prevent jest tests from failing

### ISSUE THREAD

**Issue by JamesBurnside** [frassinier/storybook-docs-toc#11]
After upgrading to storybook 6.3 our jest tests fail due to this plugin:

```bash
  ● storybook snapshot tests › encountered a declaration exception

    Jest encountered an unexpected token

    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.

    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".

    Here's what you can do:
     • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/en/ecmascript-modules for how to enable it.
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/en/configuration.html

    Details:

    D:\repos\communication-ui-sdk\common\temp\node_modules\.pnpm\@storybook\addon-docs@6.3.6_f8a667bc03ed5ec14d4d4ae3666cf073\node_modules\@storybook\addon-docs\blocks.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){import deprecate from 'util-deprecate';
                                                                                             ^^^^^^

    SyntaxError: Cannot use import statement outside a module

      at Runtime.createScriptFromCode (../../common/temp/node_modules/.pnpm/jest-runtime@26.6.3_ts-node@9.1.1/node_modules/jest-runtime/build/index.js:1350:14)
      at Object.<anonymous> (../../common/temp/node_modules/.pnpm/storybook-docs-toc@1.6.0_a91107c8ad52a4f7fdbbe084d7c56c9a/node_modules/storybook-docs-toc/dist/components/DocsContainerHOC.js:10:15)
```

This is because this plugin references to `@storybook/addon-docs/blocks`. E.g. this error originates here: https://github.com/frassinier/storybook-docs-toc/blob/a4e94b345a27d4188493aa13217f727294fef5b4/src/components/DocsContainerHOC.tsx#L5

It should now just use `@storybook/addon-docs` as the import path so jest can use the CJS bundles of addon-docs and not try to use the ESM modules.

More context on why:
* https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated-addon-knobs
* https://github.com/storybookjs/storybook/issues/13478#issuecomment-844285470

**Comment by JamesBurnside** [comment]
For anyone else who hits this issue inside Jest, a quick workaround is to use Jest's [`moduleNameMapper`](https://jestjs.io/docs/configuration#modulenamemapper-objectstring-string--arraystring).

Using this we can tell jest to map references to `@storybook/addon-docs/blocks` to instead be `@storybook/addon-docs`:

In your jest config:
```js
// jest.config.js

module.exports = {
  ...
  moduleNameMapper: {
    '@storybook/addon-docs/blocks': '@storybook/addon-docs'
  },
  ...
}
```

**Comment by frassinier** [comment]
Hey @JamesBurnside! Sorry about the delay and thanks for the workaround! 
I've just released the 1.6.1 which includes that import
